lts.json

```diff
@@ -2,13 +2,13 @@
   "autoprefixer": "~0.6.0",
   "broccoli-asset-rev": "^2.4.2",
```

Comment:
What do you mean? We are expecting that an install of ember-cli will already bring this addon?

I think we might want to list them even if ember-cli will bring them in, because we will get the latest fixed version and not the version with the `^`